lib/tests/define.rs

```diff
@@ -988,7 +988,7 @@ async fn define_statement_search_index() -> Result<(), Error> {
 			events: {},
 			fields: {},
 			tables: {},
-			indexes: { blog_title: 'DEFINE INDEX blog_title ON blog FIELDS title SEARCH english BM25(1.2,0.75,100) HIGHLIGHTS' },
+			indexes: { blog_title: 'DEFINE INDEX blog_title ON blog FIELDS title SEARCH english BM25(1.2f,0.75f,100) HIGHLIGHTS' },
```

Comment:
The extraneous `f` suffixes will go away once indexes stop storing `Number`'s i.e. when https://github.com/surrealdb/surrealdb/pull/1899/files#diff-f7d936d42096453ddf9df0148e783b8a9f8eee8627890efb64245ba86eb42811L14-R17 is merged